kalite/version.yaml

```diff
@@ -0,0 +1,397 @@
+
+"VERSION" :

```

Comment:
@aronasorman, 

Yep, I think that works! Will try once I arrive home :+1: 

I'm doing the substitution once I load the YAML file. (So it uses the VERSION value). If you think it is not necessary, sounds good to me and then, we just write the version directly on the yaml file! :+1: